fuzz-testing/src/main/scala/org/apache/comet/fuzz/Meta.scala

```diff
@@ -65,8 +65,12 @@ object Meta {
     (DataTypes.StringType, 0.2),
     (DataTypes.BinaryType, 0.1))
 
-  private def createFunctionWithInputTypes(name: String, inputs: Seq[SparkType]): Function = {
-    Function(name, Seq(FunctionSignature(inputs)))
+  private def createFunctionWithInputTypes(
+      name: String,
+      inputs: Seq[SparkType],
+      varArgs: Boolean = false): Function = {
+    Function(name, Seq(FunctionSignature(inputs, varArgs)))
```

Comment:
Should there be a check here to ensure that the `inputs` array has a single element if `varArgs == true`?  See my other comment which is related to this.